docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,160 @@
+# Bitbucket integration
+
+!!! note
+    <!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse's Bitbucket integration is new and still in Beta:
+
+    -   For now, we only show the [Accelerate metrics](../metrics/accelerate.md) using your merged pull requests to automatically detect deployments.
+    -   <!--NOTE Limitation from Bitbucket integration V1-->At the moment, we're only collecting new data, so you won’t see any metrics after completing the integration setup. We'll automatically import historical data and backfill your charts within a few weeks.
+
+<!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse integrates with Bitbucket Cloud to receive data about changes and deployments, necessary to calculate the metrics:
+
+-   [Lead time for changes](../metrics/accelerate.md#lead-time-for-changes)
+
+-   [Deployment frequency](../metrics/accelerate.md#deployment-frequency)
+
+## Setting up the Bitbucket integration
+
+!!! note
+    -   The Bitbucket integration connects to a specific [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). You must be the **owner** of that Bitbucket workspace to proceed with the integration setup.
+    -   [See below](#bb-permissions) the permissions that Pulse requires from your Bitbucket account.  
+
+To set up the Bitbucket integration:
+
+1.  On Pulse, [expand **Integrations** and select **Bitbucket**](https://app.pulse.codacy.com/integrations/bitbucket){: target="_blank"}.
+
+1.  Click **Install Bitbucket App**.
+
+    ![Installing the Pulse Bitbucket App](images/bb-installing.png)
+
+1.  Choose the workspace that you want to connect the integration to and click **Complete setup** to confirm. You can only select a workspace where you have owner permissions.
+
+    ![Choosing the Bitbucket workspace](images/bb-workspace.png)
+
+1.  Wait until you get a confirmation that Pulse successfully connected the integration to Bitbucket.
+
+    ![Pulse Bitbucket integration set up successfully](images/bb-ok.png)
```

Comment:
This is more like a note than anything else. :spiral_notepad:

I realize that @claudiacarpinteiro probably adapted the naming of the screenshots from the GitHub integration ones. In the long run, it's better to have more explicit file names to avoid "second-guessing" and make maintenance easier:

```suggestion
    ![Installing the Pulse Bitbucket App](images/bitbucket-installing.png)

1.  Choose the workspace that you want to connect the integration to and click **Complete setup** to confirm. You can only select a workspace where you have owner permissions.

    ![Choosing the Bitbucket workspace](images/bitbucket-workspace.png)

1.  Wait until you get a confirmation that Pulse successfully connected the integration to Bitbucket.

    ![Pulse Bitbucket integration set up successfully](images/bitbucket-ok.png)
```

We should also update the names of the GitHub integration later when we refresh them. :+1:

Reply:
OK, updated